content/en/agent/configuration/dual-shipping.md

```diff
@@ -327,6 +327,33 @@ DD_NETWORK_DEVICES_METADATA_ADDITIONAL_ENDPOINTS="[{\"api_key\": \"apiKey2\", \"
 
 {{% agent-dual-shipping %}}
 
+## Network Path
```

Comment:
```suggestion
## Network path
```

Reply:
We usually use "Network Path" with capital case, like in this doc: https://docs.datadoghq.com/network_monitoring/network_path/

> Network Path illustrates the route that network traffic follows from its origin to its destination.

I think it's better to keep consistent. WDYT?

Reply:
Yes, using capital case to keep it consistent with other docs is a good idea. Sorry I didn't check the other docs earlier!